pkg/ccl/backupccl/backup_planning.go

```diff
@@ -295,6 +295,9 @@ func getURIsByLocalityKV(to []string, appendPath string) (string, map[string]str
 			)
 		}
 		if localityKV == defaultLocalityValue {
+			// MB NOTE: I don't understand this logic. We will error out if defaultURI
+			// isn't already specified. Doesn't this imply that the order of the user
+			// submitted partitioned URI's would matter?
```

Comment:
I don't think the order really matters as much as it is that there is just one with `defaultLocalityValue`. If there are >1, regardless of what order they're in, when we see a second with the same defaultLocalityValue, we'll error out here.

Reply:
I see. I'll open a docs issue to clarify this. The docs currently say "At least one COCKROACH_LOCALITY must be the default", which seems misleading.